src/components/Button/Button.tsx

```diff
@@ -19,44 +15,14 @@ export interface Props {
    * Disables the field and prevents editing the contents
    */
   disabled?: boolean;
-  /**
-   * Toggles button that fills the full width of its container
-   */
-  fullWidth?: boolean;
-  /**
-   * Visually hide button text (but text is still accessible to assistive technology)
-   */
-  hideText?: boolean;
   /**
    * Link to URL. If href is present, the button will be rendered as an <a> element.
    */
   href?: string;
-  /**
-   * Name of SVG icon (i.e. caret-down, minus, warning)
-   */
-  iconName?: string;
-  /**
-   * Determines position of icon relative to button text.
-   * - **before** places icon before button text
-   * - **after** places icon after button text
-   */
-  iconPosition?: 'before' | 'after';
-  /**
-   * Button rendered on a dark backgorund
-   */
-  inverted?: boolean;
-  /**
-   * Loading state passed down from higher level used to trigger loader and text change
-   */
-  loading?: boolean;
   /**
    * On click handler for component
    */
   onClick?: MouseEventHandler;
-  /**
-   * Visually hidden additional instruction text to help provide screen reader users additional context. For instance, "View details" might be the visible button text, but screenReaderText might add additional instructions such as "for confirmation number C1234567"
-   */
-  screenReaderText?: string;
```

Comment:
We've been using `aria-label`, which I like because people already know it (and if they don't know it, want them to learn about it).

Reply:
We buy it! That specific language originated from another client who already had similar conventions in here; happy to change it to something that matches the HTML API in order to spread the good news of ARIA.

Reply:
Remove this and added `aria-label` instead 👍